### PR TITLE
Hello BadUSB blog

### PR DESCRIPTION
**Pull request by Rinorragi**
A blog post about badusb

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -83,3 +83,7 @@ authors:
     name: Jere Käpyaho
     email: jere.kapyaho@solita.fi
     url: http://www.coniferproductions.com
+  Rinorragi:
+    name: Joona Immonen
+    email: 2cb033c2473824d802cca09001da9025
+    url: www.solita.fi
```
Comment (woltage) [inline _config.yml]: Laita http:// eteen.

File: _posts/2015-09-18-Hello-BadUSB.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+There are plenty of ready scripts available. [Here](https://github.com/hak5darren/USB-Rubber-Ducky/wiki/Payloads) are few samples. There is also community made toolkit available. You should [check it out](http://www.ducktoolkit.com) since it allows you to generate scripts with certain payload. Toolkit also allows you to compile scripts on the internet but my precautioness didn't let me to test it out. 
+
+## Building the script for Ducky
+This was for me the trickiest part. Wiki tells you to download [this Duck Encoder](https://github.com/hak5darren/USB-Rubber-Ducky/wiki/Downloads). It was straightforwart to use.
+
+```powershell
+java -jar duckencoder.jar -i exploit.txt -o /media/microsdcard/inject.bin
+```
+
+Although there were one huge problem. It only supported US layout. The script didn't work unless you chose the correct keyboard layout from Windows first. Luckily community has solved this problem. There were newer versions of [commandline compiler](https://github.com/midnitesnake/USB-Rubber-Ducky) available. I just needed to pass new parameter for the encoder!
```
Comment (pvto) [inline _posts/2015-09-18-Hello-BadUSB.md]: there *was* one huge problem

**Comment by Rinorragi** [comment]
Is there somebody willing to review this as a whole?

**Comment by empperi** [comment]
Seems fine to me. Update date and I'll merge.

**Pull request merged**